components/script/dom/htmlinputelement.rs

```diff
@@ -432,6 +490,35 @@ impl HTMLInputElementMethods for HTMLInputElement {
     // https://html.spec.whatwg.org/multipage/#dom-input-alt
     make_setter!(SetAlt, "alt");
 
+    // https://html.spec.whatwg.org/multipage/#attr-fe-autocomplete
+    fn Autocomplete(&self) -> DOMString {
+        let element = self.upcast::<Element>();
+        let mut val = element.get_string_attribute(&local_name!("autocomplete"));
+        val.make_ascii_lowercase();
+
+        let val = DOMString::from(val.trim());
+
+        match &*val {
```

Comment:
Don't use `match` here, just check whether `val.is_empty()` is true, and use early returns.